website/blog/2024-02-20-release-2024-2-is-here/item.md

```diff
@@ -0,0 +1,102 @@
+
+### Audit logging now with field-level changes (Enterprise)
+
+Audit all changes made to models, including which fields were changed with previous and new values of the fields. The values are censored if they are sensitive (for example a password hash), however a hash of the changed value will still be logged.
+
+### Improve security through automated session binding
+
+Sessions for any users can now be bound to a specific geolocation (Continent, Country, City) or network (Autonomous System, subnet, IP address). If the session is accessed from a location/network that is different than that from which it was initially created, the session will be terminated.
+
+## Sure, there are some breaking changes
+
+For the first release of the year, we purposefully included several changes that are required for future product growth, some of which might require manual changes to your system configurations. Be sure to refer to the [Release Notes](https://goauthentik.io/docs/releases/2024.2) for instructions about any actions you might need to take. In general, we strive to reduce the number of breaking changes, and to always be transparent about the actions needed and the reasons for the changes. Please be sure to reach out to us via email to hello@goauthentik.io, or in [GitHub](https://github.com/goauthentik/authentik) or [Discord](https://discord.com/channels/809154715984199690/809154716507963434) with any questions and feedback!
+
+### Tenants renamed to brands
+
+Perhaps the biggest change is that _tenants_ (which were previously used to change branding configuration, default flows, and several other settings) have been renamed to *brands*. The term "Brands" more accurately reflect their usage; to configure branding, logos, colors, and overall login flow behavior. For more information, refer to the [Release Notes](https://goauthentik.io/docs/releases/2024.2) and to our documentation for _brands_.
+
+### **Helm chart breaking changes**
+
+The Helm Chart used for running authentik in Kubernetes has a number of breaking changes. Find out more in the [chart release notes](https://github.com/goauthentik/helm/releases/tag/authentik-2024.2.0).
+
+## More new functionality, DX, and UX enhancements
+
+Release 2024.2 contains even more; the following additional enhancements will make running and operating authentik even more valuable, no matter your role.
+
+-   ### AWS S3 file storage
+
+    Media files can now be stored on AWS S3. To learn more and get started with using S3, follow the [setup guide](https://goauthentik.io/docs/installation/storage-s3).
+
+-   ### *Pretend user exists* option for Identification stage
+
+    Previously the Identification stage would only continue if a user matching the user identifier exists. While this was the intended functionality, this release adds an option to continue to the next stage even if no matching user was found. "Pretend" users cannot authenticate nor receive emails, and don't exist in the database.
```

Comment:
Nitpick: "cannot ... nor" is a double negative.  I suggest "or" instead.

Reply:
One can use "cannot" and "nor" together, for sure, but it would be more standard to use "neither/nor"...  as in "'Pretend' users can neither authenticate nor receive emails, and don't exist in the database."